src/app/components/mnoe-invoices-list/mnoe-invoices-list.html

```diff
@@ -0,0 +1,49 @@
+<mno-widget icon="fa-file-text-o" heading="{{'mnoe_admin_panel.dashboard.invoices.widget.list.title' | translate}}" is-loading="$ctrl.invoices.loading">
+  <mno-widget-header> 
+  </mno-widget-header>
+  <mno-widget-body class="large no-padding">
+    <table st-pipe="$ctrl.callServer" st-table="$ctrl.invoices.list" class="table table-striped table-responsive">
+      <thead>
+        <tr>
+          <th class="col-lg-1" translate>mnoe_admin_panel.dashboard.invoices.widget.list.table.reference</th>
+          <th class="col-lg-2" translate>mnoe_admin_panel.dashboard.invoices.widget.list.table.customer</th>
+          <th class="col-lg-3" translate>mnoe_admin_panel.dashboard.invoices.widget.list.table.period</th>
+          <th class="col-lg-2" ng-if="$ctrl.isPaymentEnabled" translate>mnoe_admin_panel.dashboard.invoices.widget.list.table.expected_payment_date</th>
+          <th class="col-lg-1" translate>mnoe_admin_panel.dashboard.invoices.widget.list.table.amount</th>
+          <th class="col-lg-2" translate>mnoe_admin_panel.dashboard.invoices.widget.list.table.status</th>
+          <th class="col-lg-1"></th>
+        </tr>
+        <tr>
+          <th>
+            <input st-search="slug" placeholder="{{'mnoe_admin_panel.dashboard.invoices.search_reference' | translate}}" class="form-control input-sm search-bar" type="text"/>
+          </th>
+        </tr>
+      </thead>
+      <tbody ng-show="!$ctrl.invoices.loading">
+        <tr ng-repeat="invoice in $ctrl.invoices.list" ng-click="$ctrl.goToInvoice(invoice.id)">
```

Comment:
Can't you use directly `ui-sref` here?

Reply:
I tried the ui-sref conventional way but did not work, seems to be ignored, so ng-click was the solution I came up with. Do you suggest any better approach? :)

Reply:
Maybe it is because it is not on a link. No worries.